prow/monitoring/alertmanager-prow_secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  alertmanager.yaml: Z2xvYmFsOgogIHJlc29sdmVfdGltZW91dDogNW0KCnJvdXRlOgogIGdyb3VwX2J5OiBbJ2pvYiddCiAgZ3JvdXBfd2FpdDogMzBzCiAgZ3JvdXBfaW50ZXJ2YWw6IDVtCiAgcmVwZWF0X2ludGVydmFsOiAyaAogIHJlY2VpdmVyOiAnc2xhY2stbm90aWZpY2F0aW9ucycKCnJlY2VpdmVyczoKLSBuYW1lOiAnc2xhY2stbm90aWZpY2F0aW9ucycKICBzbGFja19jb25maWdzOgogIC0gY2hhbm5lbDogJyN0ZXN0aW5nLW9wcycKICAgIGFwaV91cmw6ICd7eyBhcGlfdXJsIH19JwogICAgaWNvbl91cmw6IGh0dHBzOi8vYXZhdGFyczMuZ2l0aHVidXNlcmNvbnRlbnQuY29tL3UvMzM4MDQ2MgogICAgdGV4dDogJ3t7IHRlbXBsYXRlICJjdXN0b21fc2xhY2tfdGV4dCIgLiB9fScKCnRl
```

Comment:
I think it would be better to have these two configs in the repo and create the secret from them. Changing this base64 data when someone wants to change the repeat interval seems silly. If there is no `k8s_rules` item for secrets, can we just shell out to `kubectl`?